modelplugin/Junos-19.3R1.8/modelmain.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build modelplugin
+
+// A plugin for the YGOT model of Junos-19.3R1.8.
+package main
+
+import (
+	"fmt"
+	"github.com/onosproject/onos-config/modelplugin/Junos-19.3R1.8/junos_19_3R1_8"
+	"github.com/openconfig/gnmi/proto/gnmi"
+	"github.com/openconfig/goyang/pkg/yang"
+	"github.com/openconfig/ygot/ygot"
+)
+
+type modelplugin string
+
+const modeltype = "Junos"
+const modelversion = "19.3R1.8"
+const modulename = "junos.so.19.3R1.8"
+
+var modelData = []*gnmi.ModelData{
+    {Name:"junos-conf-interfaces",Organization:"Juniper",Version:"2019-01-01"},
```

Comment:
shouldn't `junos-conf-system` be in here too - the same 2 as are listed in the ".env" file

Reply:
thanks - yes  there was a missing `;` in the  ".env" file